src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -75,12 +82,36 @@
   background-color: var(--pf-c-dropdown__toggle--BackgroundColor);
   border: none;
 
+  &.pf-m-split-button {
+    padding: 0;
+
+    > * {
+      padding-top: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingTop);
+      padding-right: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingRight);
+      padding-bottom: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingBottom);
+      padding-left: var(--pf-c-dropdown__toggle--m-split-button--child--PaddingBottom);
+
+      &:last-child {
+        --pf-c-dropdown__toggle--m-split-button--child--PaddingRight: var(--pf-c-dropdown__toggle--m-split-button--last-child--PaddingRight);
+      }
+    }
+
+    .pf-c-dropdown__toggle-check {
```

Comment:
The check visually is off center vertically (needs to be moved up). I know there were previous issues with the check and we had to manually move it up - so I think this is also the case here.